ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -159,6 +159,205 @@ objects:
+            #!/bin/bash
+            set -x
+            RUNNING_ETCD_POD=$(crictl pods -q --label k8s-app=etcd --state=Ready)
+            RUNNING_ETCD_CONTAINER=$(crictl ps --pod ${RUNNING_ETCD_POD} --name etcd-member -q)
+            crictl exec ${RUNNING_ETCD_CONTAINER} /bin/sh -c 'source /run/etcd/environment && ETCDCTL_API=3 etcdctl --cert /etc/ssl/etcd/system:etcd-peer:${ETCD_DNS_NAME}.crt --key /etc/ssl/etcd/system:etcd-peer:${ETCD_DNS_NAME}.key --cacert /etc/ssl/etcd/ca.crt snapshot save /var/lib/etcd/snapshot.db'
+        EOF
+          chmod +x /tmp/etcd_backup.sh
+          scp -o StrictHostKeyChecking=no -o ProxyCommand="ssh -A -o StrictHostKeyChecking=no -o ServerAliveInterval=30 -W %h:%p core@${BASTION_HOST}" /tmp/etcd_backup.sh core@${FIRST_MASTER}:/tmp
+          bastion_ssh core@${FIRST_MASTER} "sudo -i /tmp/etcd_backup.sh"
+
+          echo "Backup manifests"
+          mapfile -t MASTERS < <(oc get node -l node-role.kubernetes.io/master= -o name | cut -d '/' -f 2)
+          for master in ${MASTERS[@]}
+          do
+            bastion_ssh core@${master} "sudo -i mkdir /etc/kubernetes/manifests-backup && sudo -i cp -rvf /etc/kubernetes/manifests/* /etc/kubernetes/manifests-backup"
+          done
+
+          # echo "Upgrade to latest CI release"
+          # oc adm upgrade --to-image "${RELEASE_IMAGE_LATEST}" --force
+          # # Wait for upgrade to start
+          # oc wait clusterversion/version --for=condition=Progressing
+          # for i in $(seq 0 120); do
+          #   STATUS=$(oc get clusterversion/version -o jsonpath='{.status.conditions[?(@.type=="Progressing")]}')
+          #   if [[ "${STATUS}" == "False" ]]; then break; fi
+          #   sleep 30
+          # done
+
+          echo "Update rollback-test machineconfig"
+          oc patch machineconfig 99-rollback-test -n openshift-machine-api --patch '{"spec":{"config":{"storage":{"files":[{"contents":{"source":"data:,B","verification":{}},"filesystem":"root","mode":420,"path":"/etc/rollback-test"}]}}}}' --type=merge
+          wait_for_machineconfigpool_to_apply
+
+          echo "Distribute snapshot across all masters"
+          mapfile -t MASTERS < <(oc get node -l node-role.kubernetes.io/master= -o name | cut -d '/' -f 2)
+          for master in ${MASTERS[@]}
+          do
+            scp -o StrictHostKeyChecking=no -o ProxyCommand="ssh -A -o StrictHostKeyChecking=no -o ServerAliveInterval=30 -W %h:%p core@${BASTION_HOST}" ${KUBE_SSH_KEY_PATH} core@${master}:/home/core/.ssh/id_rsa
+            bastion_ssh core@${master} "chmod 0600 /home/core/.ssh/id_rsa"
+            bastion_ssh core@${FIRST_MASTER} "scp -o StrictHostKeyChecking=no /var/lib/etcd/snapshot.db core@${master}:/tmp/snapshot.db"
+          done
+
+          echo "Collect etcd names"
+          ETCD_CONN_STRING_TEMP=""
+          for master in ${MASTERS[@]}
+          do
+            ETCD_NAME="etcd-member-$(bastion_ssh core@${master} hostname -f)"
```

Comment:
nit: `core@${master}` -> `"core@${master}"` to make shell linters happy and on the off-chance that we got a space in a `MASTERS` entry.

Reply:
Done in 91236faa